tests/e2e_gcs_test.go

```diff
@@ -105,17 +27,17 @@ func TestCacheGCS(t *testing.T) {
 		t.Fatalf("Error unmarshalling: %v", err)
 	}
 
-	tc := tektonClient(t)
+	tc := client.TektonClient(t)
 
-	_ = tc.PipelineRuns(defaultNamespace).Delete(ctx, pr.GetName(), metav1.DeleteOptions{
-		PropagationPolicy: &deletePolicy,
+	_ = tc.PipelineRuns(resources.DefaultNamespace).Delete(ctx, pr.GetName(), metav1.DeleteOptions{
+		PropagationPolicy: &resources.DeletePolicy,
 	})
 
-	if _, err = tc.PipelineRuns(defaultNamespace).Create(ctx, pr, metav1.CreateOptions{}); err != nil {
+	if _, err = tc.PipelineRuns(resources.DefaultNamespace).Create(ctx, pr, metav1.CreateOptions{}); err != nil {
 		t.Fatalf("Error creating PipelineRun: %v", err)
 	}
 
-	if err := waitForPipelineRun(ctx, tc, pr, Succeed(pr.Name), waitInterval); err != nil {
+	if err := resources.WaitForPipelineRun(ctx, tc, pr, resources.Succeed(pr.Name), resources.WaitInterval); err != nil {
```

Comment:
nit: https://github.com/tektoncd/pipeline/blob/main/test/wait.go those exists I think. They are however not extracted from `test` package, so it may be a pain to import.